examples/using-MDX/gatsby-browser.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's Browser APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/browser-apis/
+ */
```

Comment:
Since there isn't anything in this file, you could follow the comment and delete it from the example.